scripts/update_docs.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+set -x
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DIR=$(cd $SCRIPTS_DIR; cd ..; pwd)
+VERSION=$1
+TOC_NAME=$(echo ${VERSION}toc|sed s/\\./-/g)
+
+read -r -d '' CONFIG_VERSION_BLOCK << EOM
+latest: v.*
+versions:
+- master
+EOM
+
+read -r -d '' NEW_VERSION_BLOCK << EOM
+latest: ${VERSION}
+versions:
+- master
+- ${VERSION}
+EOM
+
+read -r -d '' OLD_SCOPE_BLOCK << EOM
+  - scope:
+      path: docs\/master
+    values:
+      version: master
+      gh: https:\/\/github.com\/heptio\/sonobuoy\/tree\/master
+      layout: \"docs\"
+EOM
+
+read -r -d '' NEW_SCOPE_BLOCK << EOM
+  - scope:
+      path: docs\/master
+    values:
+      version: master
+      gh: https:\/\/github.com\/heptio\/sonobuoy\/tree\/master
+      layout: \"docs\"
+  - scope:
+      path: docs\/${VERSION}
+    values:
+      version: ${VERSION}
+      gh: https:\/\/github.com\/heptio\/sonobuoy\/tree\/${VERSION}
+      layout: \"docs\"
+EOM
+
+OLD_TOC_BLOCK="master: master-toc"
+read -r -d '' NEW_TOC_BLOCK << EOM
+master: master-toc
+${VERSION}: ${TOC_NAME}
+EOM
+
+OLD_FOOTER_BLOCK="cta_url: \/docs\/v.*"
+NEW_FOOTER_BLOCK="cta_url: \/docs\/${VERSION}"
+
+if [ -z "${VERSION}" ]
+then
+        echo "Should be called with version as first argument. No argument given, not creating docs."
+else
+    docker run --rm \
+        -v ${DIR}:/root \
```

Comment:
Running this locally creates the volume mount `/home/bridget/go/src/github.com/heptio:/root`. This results in future steps failing as the paths are incorrect. For example, `/root/site/docs/${VERSION}` should be `/root/sonobuoy/site/...`

Reply:
I think it's because I'm getting an error for `BASH_SOURCE`: `./scripts/update_docs.sh: 6: ./scripts/update_docs.sh: Bad substitution`. Is this because it's being interpreted by `/bin/sh` rather than bash? Can we use `$0` instead to refer to the first argument to the script which the script name?

Reply:
I can confirm that replacing `${BASH_SOURCE[0]}` with `$0` fixes it.

Reply:
Really? I'm getting `.../heptio/sonobuoy` whether I'm in the sonobuoy directory or the scripts directory.

Mind modifying the script a bit just for debug purposes and letting me know what the output is?

```
SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
DIR=$(cd $SCRIPTS_DIR; cd ..; pwd)

# Could you add these?
echo $SCRIPTS_DIR
echo $DIR
dirname "${BASH_SOURCE[0]}"
```

Reply:
Thanks for debugging it; I didnt see your most recent updates when I made my comment. Let me try your solution on my mac and see if that works just as well.

Reply:
Confirmed it works just fine with your suggested edit. Updated.